app/controllers/discover.js

```diff
@@ -121,6 +127,39 @@ export default ApplicationController.extend({
         });
     },
 
+    transformTypes(obj) {
+        if (typeof (obj) !== 'object') {
+            return obj;
+        }
+        var keys = Object.keys(obj);
+        var n = keys.length;
```

Comment:
[`let` is preferred to `var`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let)

`let` controls scope properly, `var` generally ends up being an unintended global variable.